### ISSUE TITLE
Bug with node 12.13.x

### ISSUE THREAD

**Issue by deleonio** [antfu-collective/unconfig#11]
![image](https://user-images.githubusercontent.com/6279703/144572343-0637dc6f-063c-4704-8dbc-77636eb6272a.png)

Can we fix that?

**Comment by antfu** [comment]
Yes, and PR welcome. But any blocker to upgrade to the latest minor version?

**Comment by sxzz** [comment]
Closing this since Node 12 reached end-of-life a long time ago.
